README.md

```diff
@@ -56,6 +56,11 @@ npm ci --ignore-scripts && npm run postinstall
 pip install -e .
 ```
 
+NOTE: If you are on Windows, make sure to first set `npm`'s default `script-shell` to `powershell`.
+```
+npm config set script-shell powershell
+```
```

Comment:
Include this in the bash snippet above and mention it is only relevant on WIndows in an inline comment? That way it becomes even more apparent that this is only something developers would need to do (by being part of that paragraph).